.github/workflows/plugins-ci.yml

```
@@ -4,25 +4,34 @@ on:
   workflow_call:
 
 jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Install Dependencies
+        run: npm i --ignore-scripts
+      - name: Lint code
+        run: npm run lint --if-present
```

the parameter needs to be added

```suggestion
        run: npm run lint --if-present=${input.lint}
```

---

The `lint` input has been removed, so don't think this is needed

---

added back the lint input in the last iteration.